docs/guide/known-limitations.md

```diff
@@ -53,6 +53,7 @@ The inconsistencies are due to:
 | Coercion of explicit arguments                       | VARP(2, 3, 4, TRUE(), FALSE(), "1",)                                      | 1.9592, based on the behavior of Excel.                                                                                                                                                                        | GoogleSheets implementation is not consistent with the standard (see also VAR.S, STDEV.P and STDEV.S function.)                  | 1,9592                                                            |
 | Ranges created with `:`                              | A1:A2<br><br>A$1:$A$2<br><br>A:C<br><br>1:2<br><br>Sheet1!A1:A2           | Allowed ranges consist of two addresses (A1:B5), columns (A:C) or rows (3:5).<br>They cannot be mixed or contain named expressions.                                                                            | Everything allowed.                                                                                                              | Same as Google Sheets.                                            |
 | Formatting inside the TEXT function                  | TEXT(A1,"dd-mm-yy")<br><br>TEXT(A1,"###.###”)                             | Not all formatting options are supported,<br>e.g. only some date formatting options: (`hh`, `mm`, `ss`, `am`, `pm`, `a`, `p`, `dd`, `yy`, and `yyyy`).<br><br>No currency formatting inside the TEXT function. | A wide variety of options for string formatting is supported.                                                                    | Same as Google Sheets.                                            |
+| Enabling array arithmetic                  | =ARRAYFORMULA(A2:A5*B2:B5)                             | The ARRAYFORMULA function enables array arithmetic. | Same as HyperFormula.                                                                    | Array arithmetic is enabled by default.                                            |
```

Comment:
HF also has an option to enable it by default like Excel does

Reply:
@wojciechczerniak done: [commit](https://github.com/handsontable/hyperformula/pull/741/commits/4d2a5be005e2093da7eb76522332c765e15c1d8a#diff-5109f9e7159c2bfac5813130c5d5bb6c9208c320ea8f47f030f009eb0071254eR56)